packages/redot-parse/index.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line n/file-extension-in-import
```

Comment:
We use TypeScript for this check. IMO this rule can be disabled project-wide.

Reply:
Good idea, will do